subprojects/packagefiles/libebml/meson.build

```diff
@@ -0,0 +1,88 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+#hacks to fix compilation with GCC11. They go away with the next version
+run_command('sed', 's/cassert/limits/g', '-i', 'src/EbmlString.cpp', check : true)
+run_command('sed', 's/cassert/limits/g', '-i', 'src/EbmlUnicodeString.cpp', check : true)
+
+#hacks to fix MSVC compilation
+run_command('sed', 's/uint64 UnixEpochDelay;/uint64 UnixEpochDelay = 978307200;/g', '-i', 'ebml/EbmlDate.h', check : true)
+run_command('sed', 's/const uint64 EbmlDate::UnixEpochDelay = 978307200;//g', '-i', 'src/EbmlDate.cpp', check : true)
```

Comment:
This cross references to https://github.com/Matroska-Org/libebml/issues/91

This seems like a relevant point made in that issue:
> when compiled with BUILD_SHARED_LIBS set these static variables are not exported, even when using EBML_DLL_API in their definition.

Wouldn't this be a better solution? I'm highly skeptical of sed-patching the upstream sources with unmerged changes.

Reply:
https://github.com/Matroska-Org/libebml/pull/94 fixes it.

With static_library, this is not needed I think.